### PR TITLE
Fixing pip & python failures

### PR DESCRIPTION
**Pull request by yogesh-mane**
Signed-off-by: yogesh-mane <ymane@redhat.com>

Fixes pip failure & python2 failures

### CODE REVIEW

**Comment by rakeshgm** [comment]
Travis ci check  is failing. Pls look into it.

File: tests/cephfs/cephfs_utils.py [hunk 1]
```diff
@@ -56,16 +56,29 @@ def get_clients(self, build):
                 out, rc = node.exec_command(cmd='sudo rpm -qa | grep -w attr')
                 output = out.read().decode()
                 output.split()
+                distro, rc =node.exec_command(cmd='sudo lsb_release -a')
```
Comment (rakeshgm) [inline tests/cephfs/cephfs_utils.py:59]: you don't need to use this command here. you can use ceph module to get the distro information like this 
https://github.com/red-hat-storage/cephci/blob/master/tests/misc_env/install_prereq.py#L44

File: tests/cephfs/cephfs_utils.py [hunk 2]
```diff
@@ -56,16 +56,29 @@ def get_clients(self, build):
...
+                                long_running=True)
+                else:
+                    node.exec_command(cmd='sudo yum install -y python2-pip')
                 if 'attr' not in output:
                     node.exec_command(cmd='sudo yum install -y attr')
                 node.exec_command(cmd='sudo yum install -y gcc python2-devel', check_ec=False)
 
                 out, rc = node.exec_command(
-                    cmd='sudo pip  list --format=columns')
+                    cmd='sudo pip2  list --format=columns')
```
Comment (rakeshgm) [inline tests/cephfs/cephfs_utils.py:77]: i would suggest stop using python2 code here. and start using python3. convert the existing scripts to py3. we wont be needing to use py2 anymore. if python3 is not installed in rhel 7, pls install

File: tests/cephfs/cephfs_utils.py [hunk 3]
```diff
@@ -1111,6 +1124,7 @@ def network_disconnect(self, ceph_object):
         nw_disconnect.write(script)
         nw_disconnect.flush()
         log.info('Stopping the network..')
+        node.exec_command(cmd='sudo yum install -y python2')
```
Comment (rakeshgm) [inline tests/cephfs/cephfs_utils.py:1127]: same as above comment. stop using python2 and start installing python3 here.

**Comment by yogesh-mane** [comment]
Hi Rakesh,

We have so much py3 failures in fs suites, which would take very long
periods of time.

Sharing a doc regarding this estimation.
https://docs.google.com/document/d/1gerhrWW88HuHvww2EhlhlmtvwZ6BzakZaa4HJXTOpM0/edit

**Comment by rakeshgm** [comment]
@yogesh-mane please fix the travis-ci failures.

File: tests/cephfs/cephfs_utils.py (hunk 1 above)
Comment (rakeshgm) [reply tests/cephfs/cephfs_utils.py:59]: @yogesh-mane any update on this. lets merge this as quickly as possible. some of the inetrop suites are failing.

**Comment by yogesh-mane** [comment]
@XDsubhash , will post logs for 4.1-rhel-8 &  4.1-rhel-7 ,still running .
3.3 suite is failing for installation , @harishmunjulur is working on it

**Comment by yogesh-mane** [comment]
@XDsubhash , will post logs for 4.1-rhel-7 & 4.1-rhel-7.
3.3 is failing for installation , @harishmunjulur is working on it.

**Comment by yogesh-mane** [comment]
@rakeshgm , will work on travis-ci failures

**Comment by yogesh-mane** [comment]
Sorry by mistake closed this pr, reopening it

**Comment by yogesh-mane** [comment]
4.1-rhel-8 logs =http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1597283254415/

**Comment by rakeshgm** [comment]
@XDsubhash if you dont have any more changes to suggest, lets get this PR merged.

**Comment by XDsubhash** [comment]
@yogesh-mane can you create a issue for 3.3 install failure with logs and brief info?

**Comment by XDsubhash** [comment]
logs posted branch is very old pl rebase ..hard to parse thru logs @harishmunjulur @yogesh-mane

**Comment by yogesh-mane** [comment]
3.3 logs = http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1597319874515/

**Comment by yogesh-mane** [comment]
@yogesh-mane can you create a issue for 3.3 install failure with logs and brief info? > sure @XDsubhash
